docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md

```diff
@@ -1,244 +1,200 @@
 
-### Supported System Version
+## Quick Steps to Build
 
-- Fedora 29, 30
-- CentOS 6.5, 7.5
-- Ubuntu 16.04, 18.04
-
-### Disk and Memory Suggestion
-
-When building type is **DEBUG**, an SSD disk with space more than **50GB** is suggested.
-
-And **8GB** memory or more is required.
-
-### Building Locally
-
-#### Step 1: Clone Code
+### Install Dependencies
+Please note that it requires root privileges to install packages.
 
+For CentOS, RedHat and Fedora users:
 ```bash
-bash> git clone https://github.com/vesoft-inc/nebula.git
-```
-
-#### Step 2: Install Dependencies
-
-- If you can download `OSS` packages,
-
-  ```bash
-   bash> cd nebula && ./build_dep.sh
-  ```
-
-- If you cannot download `OSS` packages.
-
-1. Install other dependencies through local sources
-
-   ```bash
-   bash> cd nebula && ./build_dep.sh N
-   ```
-
-2. Download the corresponding version of the dependency package
-
-- [Fedora 29/30](https://nebula-graph.oss-accelerate.aliyuncs.com/third-party/fedora29.tar.gz)
-- [CentOS 7.0~7.6](https://nebula-graph.oss-accelerate.aliyuncs.com/third-party/centos7.5.tar.gz)
-- [CentOS 6.5](https://nebula-graph.oss-accelerate.aliyuncs.com/third-party/centos6.5.tar.gz)
-- [Ubuntu 1604](https://nebula-graph.oss-accelerate.aliyuncs.com/third-party/ubuntu16.tar.gz)
-- [Ubuntu 1804](https://nebula-graph.oss-accelerate.aliyuncs.com/third-party/ubuntu18.tar.gz)
-
-3. Install the package
-
-```text
-tar xf ${package_name}.tar.gz
-cd ${package_name} && ./install.sh
-```
-
-#### Step 3: Update **~/.bashrc**
+$ yum update
+$ yum install -y make \
+                 m4 \
+                 git \
+                 wget \
+                 unzip \
+                 xz \
+                 readline-devel \
+                 ncurses-devel \
+                 zlib-devel \
+                 gcc \
+                 gcc-c++ \
+                 cmake \
+                 gperf \
+                 gettext \
+                 curl \
+                 redhat-lsb-core
 
-```bash
-bash> source ~/.bashrc
+# For CentOS 8+, RedHat 8+, and Fedora, you need install libstdc++-static, libasan
+$ yum install -y libstdc++-static libasan
 ```
 
-#### Step 4A: Build Debug Version
-
+For Debian and Ubuntu users:
 ```bash
-bash> mkdir build && cd build
-bash> cmake ..
-bash> make -j$(nproc)
-bash> sudo make install -j$(nproc)
+$ apt-get update
+$ apt-get install -y make \
+                     m4 \
+                     git \
+                     wget \
+                     unzip \
+                     xz-utils \
+                     curl \
+                     lsb-core \
+                     build-essential \
+                     libreadline-dev \
+                     ncurses-dev \
+                     cmake \
+                     gperf \
+                     gettext
 ```
 
-Or you can build the release version.
-
-#### Step 4B: Build Release Version and Regenerate the Makefile
+For Arch and Gentoo users, you can definitely handle all of these on your own, right?
```

Comment:
Shall we detail this section?
For Arch and Gentoo users, you can definitely handle all of these on your own, can't you?